src/docs/docs/getting-started/vscode-ssh.md

```diff
@@ -1,51 +1,61 @@
+| Sub-total       | \$16.70  | \$22.68  |
+| 40GB EBS Volume | \$1      | \$1      |
+| Total           | \$17.70  | \$23.68  |
 
 ## Prerequisites:
 
 - Download and install [Visual Studio Code](https://code.visualstudio.com/download)
 - Install the [Remote - SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) extension
-- Create an [AWS Account](https://aws.amazon.com/). You can watch this [part](https://www.youtube.com/watch?v=3hLmDS179YE&t=10552s) of the AWS Certified Cloud Practitioner course on creating an account if you need help.
-- Create an IAM user with administrative privileges. You will need your `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
-- Sign into your AWS Account using your IAM user
+- AWS Academy Account. You will need your `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` and your SSH key (`.pem` file)
 
 ## Create your virtual machine (AWS EC2):
 
-1. In the upper-right hand corner of your AWS Management Console, select a region. In this tutorial, `US East (Ohio) us-east-2` as your `Region`
-   ![](https://seneca-cdot-telescope.s3.amazonaws.com/aws-cloud9/2021-10-26+09_08_11-.png)
-2. In the upper-left hand corner of your AWS Management Console, click on `Services`. This will bring up a list of AWS Services, search for `EC2`
+1. In the upper-right hand corner of your AWS Management Console, select a region. In this tutorial, `US East (N. Virginia) us-east-1` as your `Region`
+
+   ![AWS Console Region](../../static/img/aws_console_region.png)
+
+2. In the search bar at the top, type in `ec2` and click on `EC2` to go to the EC2 Dashboard
+   ![AWS Search EC2](../../static/img/aws_search_ec2.png)
 3. Click on `Launch instances`
 
-- Step 1 - Choose an Amazon Machine Image (AMI): `Ubuntu Server 20.04 LTS (HVM), SSD Volume Type`
-  ![](https://seneca-cdot-telescope.s3.amazonaws.com/vscode-ssh/2021-10-26+12_47_51-vscode-ssh.md+-+telescope+-+Visual+Studio+Code.png)
-- Step 2 - Choose an Instance Type: `t2.medium`
-  ![](https://seneca-cdot-telescope.s3.amazonaws.com/vscode-ssh/2021-10-26+12_49_07-vscode-ssh.md+-+telescope+-+Visual+Studio+Code.png)
-- Step 3 - Configure Instance Details: Accept the defaults and proceed to the next step
-- Step 4 - Add Storage: Change the Size (GiB) from `8` to `20`
-  ![](https://seneca-cdot-telescope.s3.amazonaws.com/vscode-ssh/2021-10-26+12_51_25-vscode-ssh.md+-+telescope+-+Visual+Studio+Code.png)
+- Step 1 - Choose an Amazon Machine Image (AMI): `Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type`
+  ![AWS Linux 2 AMI](../../static/img/aws_linux_2_ami.png)
+- Step 2 - Choose an Instance Type: `r5.large`
+  ![AWS r5.large Instance Type](../../static/img/aws_instance_type.png)
+- Step 3 - Configure Instance Details:
+  - IAM role: LabInstanceProfile
+    ![AWS LabInstanceProfile IAM Role](../../static/img/aws_iam_role.png)
+  - Copy and paste the [aws-userdata.sh](../../../../tools/aws-userdata.sh) script into the `User data` field
```

Comment:
Question: why not include as a file?

Reply:
I thought it'd be easier to copy and paste but using a file would yield the same result. I figured it would take more steps to download the file from the repo, save it to a folder, then upload it... I use Windows... there's probably a better way to do this on Mac XD